.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @get-bridge/frontend
```

Comment:
FYI this team isn't present in this repo. https://github.com/get-bridge/translation-sync/settings/access